packages/ibm-products/src/components/CreateFullPage/CreateFullPageStep.tsx

```diff
@@ -50,15 +136,15 @@ export let CreateFullPageStep = forwardRef(
 
       // Collect any other property values passed in.
       ...rest
-    },
-    ref
+    }: CreateFullPageStepProps,
+    ref: ForwardedRef<HTMLDivElement>
   ) => {
-    const stepsContext = useContext(StepsContext);
+    const stepsContext = useContext(StepsContext) as any;
```

Comment:
Instead of any, please create or refactor the type `StepsContextType` from `CreateTearsheet.tsx` file. I think we can reuse the same interface by moving it to a common types folder. If you need any help let me know.